crowbar_framework/config/routes.rb

```diff
@@ -75,6 +75,8 @@
   post "utils/repositories/sync", controller: "repositories", action: "sync", as: "sync_repositories"
   post "utils/repositories/activate", controller: "repositories", action: "activate", as: "activate_repository"
   post "utils/repositories/deactivate", controller: "repositories", action: "deactivate", as: "deactivate_repository"
+  post "utils/repositories/activate_all", controller: "repositories", action: "activate_all", as: "activate_all_repositories"
+  post "utils/repositories/deactivate_all", controller: "repositories", action: "deactivate_all", as: "deactivate_all_repositories"
```

Comment:
Maybe it's even better to update the routes to something like:

```ruby
get "utils/repositories/activate/all", controller: "repositories", action: "activate_all", as: "activate_all_repositories"
get "utils/repositories/activate/:platform/:repo", controller: "repositories", action: "activate", as: "activate_repository"
get "utils/repositories/deactivate/all", controller: "repositories", action: "deactivate_all", as: "deactivate_all_repositories"
get "utils/repositories/deactivate/:platform/:repo", controller: "repositories", action: "deactivate", as: "deactivate_repository"
```

Than you can even drop the check if the required params are set.